Sources/Swift/MetricKit/SentryMXManager.swift

```diff
@@ -27,6 +27,12 @@ import MetricKit
 @available(tvOS, unavailable)
 @available(watchOS, unavailable)
 @objcMembers public class SentryMXManager: NSObject, MXMetricManagerSubscriber {
+    
+    let disableCrashDiagnostics: Bool
+    
+    public init(disableCrashDiagnostics: Bool = true) {
+        self.disableCrashDiagnostics = disableCrashDiagnostics
+    }
```

Comment:
m: We never use `disableCrashDiagnostics` as false in production, should we really have this option?

Reply:
I would like to keep it so that when making changes to MetricKit I can validate the stacktraces quickly. It's not much code. If you have a better idea how to achieve this let me know, and I'm going to change it.